ompi/proc/proc.h

```diff
@@ -81,6 +81,8 @@ struct ompi_proc_t {
      * actually stored in the RTE
      */
     char*                           proc_hostname;
+
+    char padding[32];         /* for future extensions (OSHMEM uses this area also)*/
```

Comment:
I think there's still an open question to @igor-ivanov as to why this value is 32 on v1.10 and it is 16 on master.